setup.cfg

```diff
@@ -25,9 +25,9 @@ install_requires =
     pandas>=1.0.3
     scikit-learn>=0.22.1
     numba>=0.48.0
-    triqler>=0.3.0
     matplotlib>=3.1.3
     lxml>=4.6.2
+    triqler @ git+ssh://git@github.com/statisticalbiotechnology/triqler.git
```

Comment:
I don't like having a non-PyPI dependency here, although it seems to make a large difference. I'll look into why this update isn't on PyPI yet.